modules/ding_user/ding_user.install

```diff
@@ -30,6 +31,18 @@ function ding_user_create_entity_cache_table() {
 }
 
 /**
+ * Creates the ding_private_key variable.
+ */
+function ding_user_create_private_key() {
+  // variable may be set in local.settings.php, but not in db.
+  if ($key = variable_get('ding_private_key', 0)) {
+    variable_set('ding_private_key', $key);
+    return;
+  }
+  variable_set('ding_private_key', drupal_get_private_key());
+}
```

Comment:
The logic here seems convoluted with unnecessary if-statements and early return.

```php
function ding_user_create_private_key() {
  // variable may be set in local.settings.php, but not in db.
  $key = variable_get('ding_private_key', drupal_get_private_key()));
  variable_set('ding_private_key', $key);
}